openbas-front/src/admin/components/atomic_testings/atomic_testing/AtomicTestingHeader.tsx

```diff
@@ -75,7 +75,7 @@ const AtomicTestingHeader = () => {
           </Button>
         ) : (
           <Button
-            style={{ marginRight: 20 }}
+            style={{ marginRight: 20, paddingTop: 5 }}
```

Comment:
i think, the problem come from the lingheight
you can use     line-height: normal; to resolve it